omod/src/main/webapp/portlets/patientHeader.jsp

```diff
@@ -19,7 +19,15 @@
 
 <%-- Header showing preferred name, id, and treatment status --%>
 <div id="patientHeader" class="boxHeader${model.patientVariation}">
-<div id="patientHeaderPatientName"><c:out value="${model.patient.personName}" /></div>
+<div id="patientHeaderPatientName">
+<c:choose>
+<c:when test="${model.showPatientDashboardLink == true && not fn:endsWith(pageContext.request.servletPath,'patientDashboardForm.jsp')}" >
```

Comment:
Now that you're using showPatientDashboardLink parameter then you shouldn't have to check of the URL ends with patientDashboardForm.jsp anymore